libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -68,6 +68,11 @@ public class AdaptiveDialog : DialogContainer
         /// </summary>
         public IRuleSelector Selector { get; set; }
 
+        /// <summary>
+        /// Gets or sets the property to return as the result ending the dialog.
+        /// </summary>
+        public string ResultProperty { get; set; } = "dialog.result";
```

Comment:
I wonder if DefaultResultProperty wouldn't be a better name?  I say this because if the dev adds an explicit EndDialog step the ResultProperty will be ignored.